### PR TITLE
Enable OptionsMenu dark mode

### PR DESCRIPTION
**Pull request by felixfbecker**
Small change for a bigger wow effect when we release.

image

### CODE REVIEW

**Review by tjkandala — APPROVED** [review]
✨

File: client/branded/src/components/SourcegraphLogo.tsx [hunk 1]
```diff
@@ -16,47 +16,47 @@ export const SourcegraphLogo: React.FunctionComponent<React.SVGAttributes<SVGSVG
...
-            fill="#242427"
+            fill="var(--sourcegraph-logo-text-color)"
```
Comment (marekweb) [inline client/branded/src/components/SourcegraphLogo.tsx:19]: This is great, a nice reminder of how flexible we can be with SVG content

**Review by marekweb — APPROVED** [review]

**Comment by felixfbecker** [comment]
Thanks for the speedy reviews!

**Review by AlicjaSuska — APPROVED** [review]
Visuals look great @felixfbecker Thank you!

**Pull request merged**